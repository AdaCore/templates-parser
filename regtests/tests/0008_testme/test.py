from test_support import *

run('testme', ["testme11.tmplt"])
run('print_tree', ["testme11.tmplt"])
