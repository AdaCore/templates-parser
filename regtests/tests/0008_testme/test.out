
   ok1

  ok cond2

[INFO] testme11.tmplt
[C_INFO]  1 FALSE
[TEXT] 
[IF] @_COND1_@
  [TEXT]    nok1
[ELSE]
  [IF] @_COND2_@
    [TEXT]    ok1
  [ELSE]
    [TEXT]    else part
  [END_IF]
[END_IF]
[TEXT] 
[IF] @_COND2_@
  [TEXT]   ok cond2
[ELSE]
  [TEXT]   nok cond2
[END_IF]
